app/gateway/roles-and-permissions.md

```diff
@@ -18,27 +18,15 @@ related_resources:
 
 Roles and permissions are administered using the {{site.base_gateway}} [RBAC system](/gateway/entities/rbac/). Roles are sets of permissions that can be assigned to admins and users and can be specific to a [Workspace](/gateway/entities/workspace). {{site.base_gateway}} uses a precedence model, from most specificity to least specificity, to determine if a user has access to an endpoint.
 
+## What are Permissions?
 
-## Default {{site.base_gateway}}  roles
+Each role may have a number of permissions that determine its ability to interact with a resource. The RBAC system provides a level of granularity that works by assigning actions on a per-resource level using the principle of least privilege.This means that a user can have **read** permissions on `/foo/bar` and **write** permissions on `/foo/bar/far`. 
 
-By default, when {{site.base_gateway}} is configured, the starting user is configured as a **Super Admin** in the `default` Workspace. Workspace's by default contain the following roles: 
+That results in the following flow: 
```

Comment:
```suggestion
```
Removing this because a new section and table display below this instead of a diagram.